spec/unit/puppet/provider/scheduled_task/win32_taskscheduler_spec.rb

```diff
@@ -20,1586 +20,1553 @@
-            'start_time'       => '14:21',
-            'schedule'         => 'once',
-            'minutes_interval' => 0,
-            'minutes_duration' => 0,
-            'enabled'          => true,
-            'index'            => 0,
-          },
-          {
-            'start_date'       => '2012-11-14',
-            'start_time'       => '15:22',
-            'schedule'         => 'once',
-            'minutes_interval' => 0,
-            'minutes_duration' => 0,
-            'enabled'          => true,
-            'index'            => 1,
-          },
-          {
-            'start_date'       => '2013-12-15',
-            'start_time'       => '16:23',
-            'schedule'         => 'once',
-            'minutes_interval' => 0,
-            'minutes_duration' => 0,
-            'enabled'          => true,
-            'index'            => 2,
-          }
-        ])
-      end
-
-      it 'should handle multiple triggers with repeat triggers' do
-        @mock_task.expects(:triggers).returns([{
-          'start_date'       => '2011-10-13',
-          'start_time'       => '14:21',
-          'schedule'         => 'once',
-          'minutes_interval' => 15,
-          'minutes_duration' => 60,
-          'enabled'          => true,
-          'index'            => 0,
-        },
-        {
-          'start_date'       => '2012-11-14',
-          'start_time'       => '15:22',
-          'schedule'         => 'once',
-          'minutes_interval' => 30,
-          'minutes_duration' => 120,
-          'enabled'          => true,
-          'index'            => 1,
-        },
-        {
-          'start_date'       => '2013-12-15',
-          'start_time'       => '16:23',
-          'schedule'         => 'once',
-          'minutes_interval' => 60,
-          'minutes_duration' => 240,
-          'enabled'          => true,
-          'index'            => 2,
-        }])
-
-        expect(resource.provider.trigger).to match_array([
-          {
-            'start_date'       => '2011-10-13',
-            'start_time'       => '14:21',
-            'schedule'         => 'once',
-            'minutes_interval' => 15,
-            'minutes_duration' => 60,
-            'enabled'          => true,
-            'index'            => 0,
-          },
-          {
-            'start_date'       => '2012-11-14',
-            'start_time'       => '15:22',
-            'schedule'         => 'once',
-            'minutes_interval' => 30,
-            'minutes_duration' => 120,
-            'enabled'          => true,
-            'index'            => 1,
-          },
-          {
-            'start_date'       => '2013-12-15',
-            'start_time'       => '16:23',
-            'schedule'         => 'once',
-            'minutes_interval' => 60,
-            'minutes_duration' => 240,
-            'enabled'          => true,
-            'index'            => 2,
-          }
```

Comment:
this block of tests, seems to have dissappeared

Reply:
it is identical to the one below